package.json

```
@@ -367,7 +367,8 @@
     "uuid": "^8.3.2",
     "valid-url": "1.0.9",
     "vm-browserify": "1.1.2",
-    "zxcvbn": "4.4.2"
+    "zxcvbn": "4.4.2",
+    "cockatiel": "3.1.1"
```

We should be using `^` ranges by default, unless we're stuck on a specific version due to a bug or incompatibility or something. 

```suggestion
    "cockatiel": "^3.1.1"
```

---

Should we change it on core repo as well? https://github.com/MetaMask/core/blob/ba88c9c0a2d86b69ccf965f2f6e0653774be5296/packages/assets-controllers/package.json#L51

---

Addressed ! Thanks For the review!!!

---

Yes, I can change it in core as well, that was a mistake